op-mode-definitions/include/ospfv3-detail.xml.i

```diff
@@ -4,6 +4,6 @@
     <help>Show detailed information</help>
   </properties>
   <!-- FRR uses ospf6 where we use ospfv3, thus alter the command -->
-  <command>vtysh -c "show ipv6 ospf6 ${@:4}"</command>
+  <command>vtysh -c "show ipv6 ospf6 $(echo ${@:4})"</command>
```

Comment:
I have to admit this fix escapes me: why does it work?

Reply:
I can't get it yet. The same question why it cut off "**detail**"?
@c-po do you have any idea?

Reply:
Ok. Needs to modify wrapper, ospfv3 => ospf6
I will do it.

Reply:
Done.